### DISCUSSION TITLE
ansible-core 2.11.0 is outdated

### DISCUSSION THREAD

**Discussion by mcantsin** [Homebrew/discussions#1716] [Casks]
Please update ansible (core) 2.11.0 to 2.11.1, as v2.11.0 has a [bug](https://github.com/ansible/ansible/pull/74562) causing ssh to connect to default port 22 (only)

**Comment by carlocab** [comment]
Please open a pull request for this. See https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request. Using `brew update-python-resources` might help -- though the `ansible` formula is a little complicated in that one can't just use it blindly. I believe the comments in the formula document how to use `update-python-resources`.
